trunk/trunk-interfaces.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package trunk;
+
+message Interface_Trunk_Description {
+  string name = 1;
+  uint32 lag_speed = 2;
+  enum LagType {
+    LagType_ROUNDROBIN = 0;
+    LagType_ACTIVEBACKUP = 1;
+    LagType_XOR = 2;
+    LagType_BROADCAST = 3;
+    LagType_8023AD = 4;
+    LagType_TLB = 5;
+    LagType_ALB = 6;
+  };
+  LagType lag_type = 3;
+  uint32 member = 4;
+  uint32 lag_id = 5;
+  bool active = 6;
+}
```

Comment:
shouldn't this be part of the ofdpa.proto?

Reply:
Im trying not to have so many definitions in the ofdpa.proto file, since this out of sync with the ofdpa-grpc stuff